### PR TITLE
LibWeb: Various simple updates/additions for ARIA roles and default roles

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change includes the following commits:

- 4ade540fc8132a3d5a6c4c28706157a712f39e63 LibWeb: Add support for the “mark” ARIA role
- 470a000f713 LibWeb: Support the “image” synonym for the “img” ARIA role
- 80ddfc9f7c9 LibWeb: Assign “dir”, “dd”, “dt” default ARIA roles per-spec
 
This change gets us passing 100% of the subtests in the `dir-role.tentative.html` and`roles.html` tests at https://wpt.fyi/results/html-aam?product=ladybird.

### CODE REVIEW

File: Libraries/LibWeb/HTML/HTMLImageElement.cpp [hunk 1]
```diff
@@ -411,10 +411,15 @@ Optional<ARIA::Role> HTMLImageElement::default_role() const
 {
     // https://www.w3.org/TR/html-aria/#el-img
     // https://www.w3.org/TR/html-aria/#el-img-no-alt
+    // https://w3c.github.io/aria/#image
+    // N.B. The “image” role value is a synonym for the older “img” role value; however, the el-img test in
```
Comment (trflynn89) [inline Libraries/LibWeb/HTML/HTMLImageElement.cpp:415]: We haven't decided to use `n.b.` over `NOTE:`, so let's not employ new comment styles here yet - let's stick with `NOTE:` for now (with hanging indents if it wraps to multiple lines).

Comment (sideshowbarker) [reply Libraries/LibWeb/HTML/HTMLImageElement.cpp]: OK — made it so

File: Libraries/LibWeb/HTML/HTMLImageElement.cpp [hunk 2]
```diff
@@ -412,13 +412,13 @@ Optional<ARIA::Role> HTMLImageElement::default_role() const
     // https://www.w3.org/TR/html-aria/#el-img
     // https://www.w3.org/TR/html-aria/#el-img-no-alt
     // https://w3c.github.io/aria/#image
-    // N.B. The “image” role value is a synonym for the older “img” role value; however, the el-img test in
-    // https://wpt.fyi/results/html-aam/roles.html expects the value to be “image” (not “img”).
+    // Note: The "image" role value is a synonym for the older "img" role value; however, the el-img test in
```
Comment (trflynn89) [inline Libraries/LibWeb/HTML/HTMLImageElement.cpp:415]: This was squashed into the wrong commit. (and we use NOTE, all capitals).

Comment (sideshowbarker) [reply Libraries/LibWeb/HTML/HTMLImageElement.cpp]: oops, thanks — fixed now

**Pull request merged**